moped-editor/src/views/projects/projectView/ProjectFiles.js

```diff
@@ -149,58 +161,94 @@ const ProjectFiles = props => {
 
   const fileTypes = ["", "Funding", "Plans", "Estimates", "Other"];
 
+  // remove the FilePond and s3 added path for display, ex:
+  // 'private/project/65/80_04072022191747_40d4c982e064d0f9_1800halfscofieldridgepwkydesignprint.pdf'
+  const cleanUpFileKey = (str) => str.replace(/^(?:[^_]*_){3}/g, "");
+
   /**
    * Column configuration for <MaterialTable>
    */
   const columns = [
     {
       title: "Name",
       field: "file_name",
-      validate: rowData => {
+      validate: (rowData) => {
         return rowData.file_name.length > 0 ? true : false;
       },
-      render: record => (
-        <Link
-          className={classes.downloadLink}
-          onClick={() => downloadFileAttachment(record?.file_key, token)}
-        >
-          {record?.file_name}
-        </Link>
-      ),
-      editComponent: props => (
+      editComponent: (props) => (
         <TextField
           variant="standard"
           id="file_name"
           name="file_name"
           value={props.value}
-          onChange={e => props.onChange(e.target.value)}
-          helperText="Required" />
+          onChange={(e) => props.onChange(e.target.value)}
+          helperText="Required"
+        />
       ),
     },
+    {
+      title: "File",
+      field: "file_url",
+      validate: (rowData) => {
+        return rowData.file_name.length > 0 ? true : false;
```

Comment:
double checking here, but `file_name` could never be null, right?

Reply:
good catch—i believe it should be possible for this to be null. this is related to my comment below about handling the input value in the edit component the same way it is handled in the form 👇